ConnectionFactory.php

```diff
@@ -84,6 +84,9 @@ private function initializeTypes()
                 Type::addType($type, $typeConfig['class']);
             }
             if ($typeConfig['commented']) {
+                @trigger_error(
+                    'The commented option of the type configuration is deprecated since DoctrineBundle 1.7.
+                    Use the feature of the Type class Instead', E_USER_DEPRECATED);
```

Comment:
Alignment is super weird, and there's a newline in the text too (with whitespace)